app.py

```diff
@@ -3,47 +3,45 @@
 from logger import *
 import os
 
-
 app = Flask(__name__)
 from models import *
+
 if app.config['ENV'] == 'production':
     app.config.from_object('config.ProductionConfig')
-if app.config['ENV'] == 'development':
+elif app.config['ENV'] == 'development':
     app.config.from_object('config.DevelopmentConfig')
 else:
     app.config.from_object('config.BaseConfig')
     
-
-
 app.logger.info(app.config)
 
 @app.route("/")
 def home():
-    return render_template("index.html")
+    return render_template("index.html", title="Eventsly", pageStyles="main {padding: 0;}")
```

Comment:
Notice these keyword arguments - `title` and `pageStyles` are referenced in 'base.html', this is how they are sent.